packages/block-library/src/image/image.js

```diff
@@ -842,6 +844,22 @@ export default function Image( {
 	const shadowProps = getShadowClassesAndStyles( attributes );
 	const isRounded = attributes.className?.includes( 'is-style-rounded' );
 
+	/**
+	 * Get the post type, post ID, and query ID from the context for featured image control.
+	 */
+	const { postType: postTypeSlug, postId, queryId } = context;
```

Comment:
```suggestion
	const { postType, postId, queryId } = context;
```

- This comment seems redundant to me, since it's pretty obvious what this code does.
- Since there are no conflicting variable names, I don't think we need to use aliases.